sar_pipeline/aws/metadata/odc.py

```diff
@@ -1,38 +1,5 @@
-import re
-
-
-def get_collection_number(collection: str) -> int:
-    """Get the collection number from the collection string
-
-    Parameters
-    ----------
-    collection : str
-        Collection of the product. e.g. s1_rtc_c1. The collection MUST end in cX where X
-        is an integer associated with the collection. E.g. rtc_s1_c1.
-
-    Returns
-    -------
-    int
-        number. e.g. s1_rtc_c1 -> 1
-
-    Raises
-    ------
-    ValueError
-        Invalid collection name.
-    """
-
-    # ensure the collection ends with cX, where X is a positive integer
-    match = re.search(r"c(\d+)$", collection)
-    if not match:
-        raise ValueError(
-            f"Invalid collection name. The collection MUST end in cX where X"
-            " is an integer associated with the collection. E.g. rtc_s1_c1."
-        )
-    return int(match.group(1))
-
-
 def get_odc_product_name(product, collection_number, polarisations):
-    """set the odc:product value. WARNING this must align with
+    """get the odc product name. WARNING this must align with
     the DEA product name at indexing into the datacube.
     These are hard-coded and set by the provided `collection_number`.
     """
```

Comment:
We discussed adding "nrb" to the product name, which will allow us to potentially index other ARD products in future. So perhaps the names below need to change to `ga_s1_nrb_iw_vv_vh_c0` etc?